src/styles/editor/editor.css

```diff
@@ -1,7 +1,7 @@
 .editor {
-  height: 100vh;
-  width: 100vw;
-  position: relative;
+  height: 100%;
```

Comment:
Why do these need to be changed to percentages?

Reply:
I've found that `100vh` isn't what someone would normally expect on iOS -- it includes the bottom toolbar such that anything with `100vh` will overflow the screen.

So then, the only safe way to use `vh` units is for something much smaller than the full screen, like `50vh`.

Reply:
Ok that sounds like something I may have to consider for the help system overlay.  Anyways I don't see anything wrong with the code so I approve of the PR.